cookbooks/aws-parallelcluster-test/recipes/tests.rb

```diff
@@ -101,6 +101,42 @@
     raise "node_type must be HeadNode or ComputeFleet"
   end
 end
+###################
+# Scheduler Plugin
+###################
+if node['cluster']['scheduler'] == 'plugin'
+  if node['cluster']['node_type'] == "HeadNode"
+    execute 'check artifacts are in target_source_path' do
+      command "sudo ls #{node['cluster']['scheduler_plugin']['home']} | grep develop"
+    end
+    execute 'check handler-env.json in path' do
+      command "ls #{node['cluster']['shared_dir']}/handler-env.json"
+    end
+    execute 'check get compute fleet script in path' do
+      command "ls /usr/local/bin/get-compute-fleet-status.sh"
+    end
+    execute 'check update compute fleet script in path' do
+      command "ls /usr/local/bin/update-compute-fleet-status.sh"
+    end
+    execute "check clusterstatusmgtd is configured to be executed by supervisord" do
+      command "#{node['cluster']['cookbook_virtualenv_path']}/bin/supervisorctl start clusterstatusmgtd"
```

Comment:
the clusterstatusmgtd must be already running in this phase, so calling a start is going go false the result of the test. You should check for the status instead, and verify that it's running, e.g. using `...bin/supervisorctl status clusterstatusmgtd`and confirming it's RUNNING

Reply:
Done